lib/v2/instructables/radar.js

```diff
@@ -0,0 +1,69 @@
+module.exports = {
+    'www.instructables.com': {
+        _name: 'Instructables',
+        all: [
```

Comment:
This will match the FQDN `all.www.instructables.com` and `circuits.www.instructables.com`.

Use
```suggestion
    'instructables.com': {
        _name: 'Instructables',
        '.': [
```
to match FQDN `www.instructables.com`